rust/codec-lexical/tests/snapshots/examples__ghost.decode.losses.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: rust/codec-lexical/tests/examples.rs
+expression: info.losses
+---
+CodeBlockNode.caption: 1
+HtmlNode.visibility: 1
+Link: 1
+List: 2
+Unknown (aside): 1
+Unknown (image): 1
```

Comment:
Need to decide how we will handle these. For `image`, `audio`, `video` and `file` Ghost Koenig cards I am leaning towards adding `ImageObject`, `AudioObject` etc to Stencila's `Block` enum, rather than wrapping in a paragraph as we currently do.